x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.tsx

```diff
@@ -639,6 +684,17 @@ export const AlertsList: React.FunctionComponent = () => {
           setPage(changedPage);
         }}
       />
+      {manageLicenseModalOpts && (
+        <ManageLicenseModal
+          licenseType={manageLicenseModalOpts.licenseType}
+          alertTypeId={manageLicenseModalOpts.alertTypeId}
+          onConfirm={() => {
+            window.open(`${http.basePath.get()}/app/management/stack/license_management`, '_blank');
```

Comment:
I'm still a Kibana UI n00b, and so a little surprised we're opening the new window like this!  Old school!  Personally, I like this experience because it doesn't "get in my way" (leaves the current tab contents intact), but I would have guessed we might had some wrapper for it, for security reasons or such.

Reply:
The manage license button inside the alert details view does the same thing, so I replicated it to give the same experience.